tests/integration/ceph_smoke_test.go

```diff
@@ -87,6 +93,10 @@ func (suite *SmokeSuite) TearDownSuite() {
 }
 
 func (suite *SmokeSuite) TestBlockCSI_SmokeTest() {
+	if RookTestOpenshift {
+		logger.Info("Skipping this test on OpenShift ENV because it's blocked by github issues: 3517 and 3520")
```

Comment:
#3517 and #3520 are closed now (merged with #3373). Can we enable this test agin?